### ISSUE TITLE
NTR: organism count and organism counting process

### ISSUE THREAD

**Issue by ramonawalls** [BiodiversityOntologies/bco#115]
## Label

organism count

syn: organism amount

## Textual definition

An amount that is measured by the number of organisms of the same taxon in a sample.

## Suggested parent term

pato: amount

## Attribution

https://orcid.org/0000-0001-8815-0078

----------
## Label

organism counting process

## Textual definition

An observing process in which the output is a data item about the number of organisms of the same taxon in the observed sample.

## Suggested parent term

observing process

## Attribution

https://orcid.org/0000-0001-8815-0078

**Comment by ramonawalls** [comment]
See earlier discussion at PopulationAndCommunityOntology/pco#86 and UA-SRC-data/srpdio#3.

**Comment by kaiiam** [comment]
@ramonawalls I presume we would intend for this new `organism count` (quality) term to be used within a DOSDP pairing it with an NCBI Taxon term? Perhaps something like the following: 

Label: `X count` where X is an NCBI Taxon term

definition: 

> An amount that is measured by the number of organisms of the taxon X in a sample.

Equivalence axiom: 
```
'organism count'
 and ('inheres in' some X)
```

**Comment by kaiiam** [comment]
If so would we want to pre-compose some examples within BCO? Projects like UA-SRC, BCODMO and Planet Microbe would all benefit form such a pattern. Should we pre-compose? E.g. certain commonly collected taxa?

Would we just set this kind of thing up at a high level within BCO and allow people to import these classes into an application ontology and or use them with a setup like GEOME? Or would we perhaps pre-compose and give iri's to new count terms paird with all of NCBITaxon? Perhaps as a seperate module in BCO? 

Would this kind of thing be helpful to an organization like [GBIF](https://www.gbif.org/)? They seem to be they using using DarwinCore's https://dwc.tdwg.org/terms/#taxonID as a property? I wonder how this is used now if they have records directing to an NCBITaxon purls? I wonder if NCBITaxon IDs would be more useful to use-cases like theirs than precomposed `BCO:x taxon count` terms. 

My understanding of the purpose of the BCO data model is to allow for reasoning and get more out of data or be able to compare different datatypes. Does precomposing help us? For the first 3 mentioned use-cases I think the answer is yes. If so then having pre-composed paired iris seems like it would make things more interoperable I guess we'd be reducing the ambiguity between a taxon itself and a measured count of such organisms? Would this help prevent punning by having the instances (rows in an abundance dataset transformed into RDF) be subclass of these named `BCO:x taxon count` classes instead of being subclass to the NCBITaxon terms themselves? 

Sorry for the long-winded message.

**Comment by ramonawalls** [comment]
1) I don't want to precompose classes for specific organisms in BCO until there is an actual demand for them, but when the specific need arises, we can add them, probably as a module as you suggest.

2)
> Label: `X count` where X is an NCBI Taxon term
> 
> definition:
> 
> > An amount that is measured by the number of organisms of the taxon X in a sample.
> 

I will add a subclass of organism count that is `organism count in specimen`. Many counts take place in situ and don't involve a specimen or material sample. 

3)
> Equivalence axiom:
> 
> ```
> 'organism count'
>  and ('inheres in' some X)
> ```
This does not quite work, because an instance X (an NCBI class) is an instance of an individual organism. I think it is better to use 'organism count and ('is about' some X). But this needs to be tested out.

**Comment by kaiiam** [comment]
Would you be willing to let me pre-compose some specific organisms in BCO for SRPDIO, Planet Microbe and or BCODMO? Or would this be better done importing these BCO classes and then making the pre-composed sub classes in the respective application ontologies? 

> `organism count in specimen` 

👍, yes sorry I was being a bit marine-centric there of course the super class wouldn't need to involve a specimen. 

> 'organism count and ('is about' some X)

That makes more sense to me. I'd be happy to try it out with a DOSDP in BCO or a respective application ontology importing these terms.

**Comment by ramonawalls** [comment]
@kaiiam Let's test out the taxon specific classes in SRPDIO, and if they work well, we can move them into BCO. Iam envisioning a BCO module that for these classes. Working on modularization for the next release.

**Comment by kaiiam** [comment]
Sounds good, I think I can prepare an appropriate DOSDP to generate this module. I presume I'll need to wait for the BCO release in order to import these `organism count` and `organism count process` terms into SRPDIO. Once imported, I can test out the module there. I believe this is in scope for my current work with BCODMO.

**Comment by ramonawalls** [comment]
I think we need to add a third term here, which is a data item called `organism count`. It will be the output of `organism counting process` and be about some `organism amount`. This means the primary label for the quality should be `organism amount`, which is also more consistent with PATO. The quality can have a value which is the count, which can be zero to infinity.  This will make it easier for modeling data.

**Comment by kaiiam** [comment]
Yes agreed, I wasn't able to anything quite like `organism count` the closest thing I could find in PATO is [amount](http://purl.obolibrary.org/obo/PATO_0000070), which I guess is on track to be to be the superclass. Let me know if you want help making these terms, and or when I could try making the DOSDP module. I'd like to use the `organism count` term in the new BCODMO semantic model I'm helping to build.
